api/app/signals/apps/my_signals/management/commands/create_my_signal_token.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.core.validators import validate_email
+from django.utils import timezone
+
+from signals.apps.my_signals.models import Token
+from signals.apps.signals.models import Reporter
+
+
+class Command(BaseCommand):
+    help = 'Create a token for a specific reporter'
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', type=str, help='E-mail address of the reporter to create a token for')
+        parser.add_argument('--refresh', action='store_true', dest='_refresh', help='Refresh the token')
+        parser.add_argument('--dry-run', action='store_true', dest='_dry_run', help='Dry-run mode')
```

Comment:
Are you sure we need the `--dry_run` mode? It does not seem to be used below at all.